### PR TITLE
Fix #572: update flash to 4.7.1

### LINKED ISSUE #572: flash-tarball update version
Need to run tests to ensure that the version is compatible with various packages we are using.

### PR DESCRIPTION
**Pull request by e-carlin**

### CODE REVIEW

**Comment by e-carlin** [comment]
I've put the latest flash source on the build machine. We'll just need to build once this is merged.

File: installers/flash-tarball/radiasoft-download.sh [hunk 1]
```diff
@@ -62,36 +62,21 @@ flash_tarball_patch_and_update_tgz() {
...
-     parseCmd.init()
-     # setup.py is FLASH_HOME/bin/setup.py
-EOF
     patch --quiet tools/python/flmake/setup.py <<'EOF'
 @@ -32,7 +32,7 @@
      global RUNTIME_FILES
- 
+
      # this is not a threaded application
 -    sys.setcheckinterval(10000)
```
Comment (robnagler) [inline installers/flash-tarball/radiasoft-download.sh:70]: Are the two setup.py's different?

**Review by robnagler — APPROVED** [review]
Seems fine to me, just one question.

File: installers/flash-tarball/radiasoft-download.sh (hunk 1 above)
Comment (e-carlin) [reply installers/flash-tarball/radiasoft-download.sh:70]: Yes, for whatever reason tools/python/flmake/setup.py has not been fixed.

**Pull request merged**